path_2023.py

```diff
@@ -0,0 +1,64 @@
+"""
+Task 1 path.
+"""
+import pathlib
+import dronekit
```

Comment:
Nit: Single empty line above this.

Reply:
What I meant is empty line between pathlib and dronekit imports. No empty line between triple quotation marks and `import pathlib` .